### PR TITLE
Post for iviewer 0.6.0

### PR DESCRIPTION
**Pull request by jburel**
Add post for iviewer 0.6.0

### CODE REVIEW

File: _posts/2018-11-16-iviewer-0-6-0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+---
+Today we are releasing version 0.6.0 of OMERO.iviewer: an OMERO.web app
+for visualizing images in OMERO.
+
+This version requires OMERO 5.4.0 or newer.
+This version includes:
+
+- cached image settings to allow navigation between images without losing unsaved settings
+- improved loading of thumbnails
+- enabled zoom using double-clicking on the image
```
Comment (rgozim) [inline _posts/2018-11-16-iviewer-0-6-0.md]: enabled double-click to zoom on image viewing

File: _posts/2018-11-16-iviewer-0-6-0.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+intro-blurb: The OME team is pleased to announce the release of OMERO.iviewer 0.6.0
+---
+Today we are releasing version 0.6.0 of OMERO.iviewer: an OMERO.web app
+for visualizing images in OMERO.
+
+This version requires OMERO 5.4.0 or newer.
+This version includes:
+
+- cached image settings to allow navigation between images without losing unsaved settings
+- improved loading of thumbnails
```
Comment (rgozim) [inline _posts/2018-11-16-iviewer-0-6-0.md]: improved thumbnail loading

**Comment by jburel** [comment]
@rgozim text updated as suggested

**Comment by sbesson** [comment]
We'll probably need to bump the date of the post

**Comment by jburel** [comment]
It has been bumped to 19.11

**Pull request merged**